app/scss/add-project-popup.scss

```diff
@@ -0,0 +1,67 @@
+@import "variables.scss";
+@import "avenir-font.scss";
+
+.c-add-project__header {
+  margin-top: 2rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+.c-add-project__close {
+  display: flex;
+  align-items: center;
+  &--cross {
+    fill: #f64747;
+    cursor: pointer;
+    &:hover {
+      box-shadow: 0 0 3rem rgba(246, 71, 71, 0.75);
+    }
+  }
+}
+.c-add-project {
+  &__popup {
+    position: relative;
+    top: -55rem;
+    width: 90rem;
+    height: 50rem;
+    border-radius: 5rem;
+    background-color: $clr-main-text;
+    box-shadow: 0 0 3rem rgba(120, 120, 120, 0.75);
+    display: none;
+    margin: auto;
+    padding: 0.5rem 2rem;
+    &--active {
+      display: block;
+      z-index: 3;
+    }
+  }
+  &__header {
+    position: relative;
+    color: $clr-para-text;
+  }
+}
+.c-add-project--form {
+  width: 70%;
+  margin-top: 2rem;
+  height: 5rem;
+  width: 70%;
+}
+.c-add-project__input {
+  font-family: "avenir", sans-serif;
```

Comment:
All font settings could be as part of a `_resets.scss` file, for example:

`
p,
input,
select {
  font-family:...
}
`